cmd/ls.go

```diff
@@ -13,24 +15,19 @@ import (
 
 func lsCommand() *cobra.Command {
 	lsCmd := &cobra.Command{
-		Use:   "ls [<path>]",
-		Short: "Lists the resources at the indicated path",
-		Args:  cobra.MaximumNArgs(1),
-		RunE:  toRunE(lsMain),
+		Use:   "ls [<path>]...",
+		Short: "Lists the children of the specified paths, or current directory if not specified",
+		Long: `Lists the children of the specified paths, or current directory if
+no path is specified. If the -l option is set, then the name,
+last modified time, and supported actions are displayed for
+each child.`,
+		RunE: toRunE(lsMain),
 	}
 	lsCmd.Flags().BoolP("long", "l", false, "List in long format")
 	return lsCmd
 }
 
-func headers() []cmdutil.ColumnHeader {
```

Comment:
Removing these makes the output cleaner with multiple paths. The built-in `ls -l` also doesn't have any headers for its long format, so I think we should be safe here.

Reply:
Makes sense. It'd be nice if `<unknown>` somehow reflected that it was expected to be a time, but not sure how to do that without being overly verbose. Maybe `<no mtime>`?

Reply:
`<mtime unknown>` ?

Reply:
I'm ok with that.